moveit_core/robot_state/include/moveit/robot_state/attached_body.h

```diff
@@ -120,6 +120,12 @@ class AttachedBody
     return subframe_poses_;
   }
 
+  /** \brief Get subframes of this object (relative to the link) */
```

Comment:
This is probably just another issue with your ongoing pose work, but why are *global* frames documented as *relative to the link*? Maybe extend/change the comment?

Reply:
I think I just forgot to update this comment, my bad.